system-tests/tests/settings-cli-modify-list

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Check that cobbler-settings can modify value from the YAML
+
+# Arrange
+source ${SYSTESTS_PRELUDE} && prepare
+
+set -x -e -o pipefail
+
+TEST_CONFIG="${tmp}/settings-list.yaml"
+EXPECTED_CONFIG="${tmp}/settings-expected.yaml"
+cat <<-END >$TEST_CONFIG
+build_reporting_email:
+  - 'root@localhost'
+END
+
+cat <<-END >$EXPECTED_CONFIG
+# Cobbler settings file
+# Docs for this file can be found at: https://cobbler.readthedocs.io/en/latest/cobbler-conf.html
+
+build_reporting_email: []
+END
+
+TEST_KEY="build_reporting_email"
+TEST_VALUE=""
```

Comment:
Not sure how the modify command works, but should this test value be a string or a list?

Reply:
According to `utils.input_string_or_list()` this should return an empty list then.